### PR TITLE
Added php8.1 support

### PR DESCRIPTION
**Pull request by konarshankar07**
|    Q          |   A
|-------------- | ------
| Documentation | no
| Bugfix        | no
| BC Break      | no
| New Feature   | no
| RFC           | no
| QA            | no

### Description

Declared php 8.1 support
https://github.com/magento/magento2/issues/34242

### CODE REVIEW

**Comment by konarshankar07** [comment]
Hello @ihor-sviziev
Any suggestion for the php8.1 update?
Thanks

**Comment by ihor-sviziev** [comment]
First of all - keep the php 7.3 and 7.4 support as there no reason for dropping support without any changes in code. After that, phpcs tests should pass

**Comment by konarshankar07** [comment]
I have readded the php 7.3 and 7.4 support

File: composer.json [hunk 1]
```diff
@@ -25,13 +25,13 @@
         }
     },
     "require": {
-        "php": "^7.3 || ~8.0.0",
+        "php": "^7.3 || ^7.4 || ~8.0.0 || ~8.1.0",
```
Comment (Xerkus) [inline composer.json]: `^7.4` is not needed, it is already in range

Comment (konarshankar07) [reply composer.json]: I have removed the 7.4 version

**Pull request merged**